src/system.c

```diff
@@ -9,6 +9,7 @@
 #include <errno.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <sys/time.h> // TODO: for #include <sys/_timeval.h>
```

Comment:
I don't understand this TODO?

Reply:
Sorry it was basically not explained

My IDE is unhappy about `struct timeval` being used in this file, saying it's not defined

I don't know why compilation still works

the man page on my system says (`man timeval`):
```
       timeval
              Include: <sys/time.h>.  Alternatively, <sys/resource.h>, <sys/select.h>, or <utmpx.h>.

              struct timeval {
                  time_t      tv_sec;  /* Seconds */
                  suseconds_t tv_usec; /* Microseconds */
              };
```

system.c has none of these includes

again idk why compilation doesn't fail

Reply:
I'd remove this then, it seems more a problem of the IDE than a libdragon issue